ui/src/services/services.js

```diff
@@ -1,11 +1,13 @@
 import http from "../http-common"
 class StudentService {
   getAll() {
+
     return http.get("/students");
 
   }
   create(data) {
-    alert("message")
+    alert("called..")
```

Comment:
move this alert after the  response is received on successfully creating a student Data.